scripts/set-env-variable/on-start.sh

```diff
@@ -9,11 +9,24 @@ set -e
 #   1. Ensure the Notebook Instance execution role has permission of SageMaker:ListTags
 #
 
+#sudo -u ec2-user -i <<'EOF'
```

Comment:
yes. option for customer to uncomment if they desire to run in child shell.

Reply:
So if user uncomment this line, then the commands run as the ec2-user. Otherwise the commands run as the root user. Have you put this difference into consideration?

Reply:
yes. i added sudo to the system restart command as a result.